cmd/admin/pause_identifier.go

```diff
@@ -0,0 +1,148 @@
+package main
+
+import (
+	"context"
+	"encoding/csv"
+	"errors"
+	"flag"
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/letsencrypt/boulder/identifier"
+	sapb "github.com/letsencrypt/boulder/sa/proto"
+)
+
+// subcommandPauseIdentifier encapsulates the "admin pause-identifiers" command.
+type subcommandPauseIdentifier struct {
+	batchFile string
+}
+
+var _ subcommand = (*subcommandPauseIdentifier)(nil)
+
+func (p *subcommandPauseIdentifier) Desc() string {
+	return "Administratively pause an account preventing it from attempting certificate issuance"
+}
+
+func (p *subcommandPauseIdentifier) Flags(flag *flag.FlagSet) {
+	flag.StringVar(&p.batchFile, "batch-file", "", "Path to a CSV file containing (account ID, identifier type, list of identifier strings)")
+}
+
+func (p *subcommandPauseIdentifier) Run(ctx context.Context, a *admin) error {
+	if p.batchFile == "" {
+		return errors.New("the -batch-file flag is required")
+	}
+
+	identifiers, err := a.readPausedAccountFile(p.batchFile)
+	if err != nil {
+		return err
+	}
+
+	_, err = a.pauseIdentifiers(ctx, identifiers)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// pauseIdentifiers allows administratively pausing a set of domain names for an
+// account. It returns a slice of PauseIdentifiersResponse or an error.
+func (a *admin) pauseIdentifiers(ctx context.Context, incoming []pauseCSVData) ([]*sapb.PauseIdentifiersResponse, error) {
+	if len(incoming) <= 0 {
+		return nil, errors.New("cannot pause identifiers because no pauseData was sent")
+	}
+
+	var responses []*sapb.PauseIdentifiersResponse
+	for _, data := range incoming {
+		req := sapb.PauseRequest{
+			RegistrationID: data.accountID,
+			Identifiers: []*sapb.Identifier{
+				{
+					Type:  string(data.identifierType),
+					Value: strings.Join(data.identifierValue, ","),
```

Comment:
I don't think this works? The SA's `PauseIdentifiers` function expects each input identifier to be a single <type, value> pair. Having the value be a comma-separated list doesn't work. You'll need to do something more like
```go
idents := make([]*sapb.Identifier, 0, len(data.identifierValue)
for _, v := range data.identifierValue {
  idents = append(idents, &sapb.Identifier{data.identifierType, v)
}
req := sapb.PauseRequest{
  RegistrationID: data.accountID,
  Identifiers: idents,
}
```
In fact, this makes me question whether it's even worthwhile allowing the input to have arbitrarily many identifiers on a line. How likely is it that the tools we use to produce lists of things to pause will produce output in that format? I'd kind of expect our most common use-case will simply be
```csv
12345,dns,example.com
12345,dns,example.org
54321,dns,foo.bar
```
rather than listing both identifiers for user 12345 on the same line. I think this would also make the csv loading simpler.